bip78/src/receiver/mod.rs

```diff
@@ -4,17 +4,28 @@ use bitcoin::{Script, TxOut};
 mod error;
 
 pub use error::RequestError;
-use error::InternalRequestError;
+use error::{InternalRequestError, CheckError};
+
+use crate::receiver::check_flags::*;
 
 pub trait Headers {
     fn get_header(&self, key: &str) -> Option<&str>;
 }
 
-pub struct UncheckedProposal {
+pub mod check_flags {
+    pub const CAN_BROADCAST_ORIGINAL_PSBT: u8 =     0b0001;
+    pub const NO_OWNED_ORIGINAL_PSBT_INPUTS: u8 =   0b0010;
+    pub const HAS_SCRIPT_TYPE_AVAILABLE: u8 =       0b0100;
+    pub const NO_SEEN_ORIGINAL_PSBT_INPUTS: u8 =    0b1000;
+}
```

Comment:
Flags feel messy in a module that needs to be `use`d. Is there a better way to organize it?